### PR TITLE
New readme

### PR DESCRIPTION
**Pull request by asinghvi17**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <img src="https://raw.githubusercontent.com/JuliaPlots/Makie.jl/master/assets/logo.png" alt="Makie.jl" width="480">
 </div>
 
-From the japanese word [_Maki-e_](https://en.wikipedia.org/wiki/Maki-e), which is a technique to sprinkle lacquer with gold and silver powder.
+From the japanese word _Maki-e_
```
Comment (briochemc) [inline README.md:5]: Removed punctuation here? (not sure intentional)

File: README.md [hunk 2]
```diff
@@ -45,99 +44,25 @@ sudo dnf install cmake libXrandr-devel libXinerama-devel libXcursor-devel
 
 # Ecosystem
 
-`Makie.jl` is the metapackage for a rich ecosystem, which consists of [`GLMakie.jl`](https://github.com/JuliaPlots/GLMakie.jl), [`CairoMakie.jl`](https://github.com/JuliaPlots/CairoMakie.jl) and [`WGLMakie.jl`](https://github.com/JuliaPlots/WGLMakie.jl) (the backends); [`AbstractPlotting.jl`](https://github.com/JuliaPlots/AbstractPlotting.jl) (the bulk of the package); and [`StatsMakie.jl`](https://github.com/JuliaPlots/StatsMakie.jl) (statistical plotting support, as in [`StatsPlots.jl`](https://github.com/JuliaPlots/StatsPlots.jl)).
+Makie is the metapackage for a rich ecosystem, which consists of [GLMakie.jl](https://github.com/JuliaPlots/GLMakie.jl), [CairoMakie.jl](https://github.com/JuliaPlots/CairoMakie.jl) and [WGLMakie.jl](https://github.com/JuliaPlots/WGLMakie.jl) (the backends); [AbstractPlotting.jl](https://github.com/JuliaPlots/AbstractPlotting.jl) (the bulk of the package); and [StatsMakie.jl](https://github.com/JuliaPlots/StatsMakie.jl) (statistical plotting support, as in [StatsPlots.jl](https://github.com/JuliaPlots/StatsPlots.jl)
```
Comment (briochemc) [inline README.md:47]: Removed `).` to close the parenthesis and the punctuation here. (Pretty sure it's a typo)

File: README.md [hunk 3]
```diff
@@ -45,99 +44,25 @@ sudo dnf install cmake libXrandr-devel libXinerama-devel libXcursor-devel
...
 
-There is experimental support for using Plots.jl and RecipesBase.jl recipes at [`MakieRecipes.jl`](https://github.com/JuliaPlots/MakieRecipes.jl), and a prototype TeX plotting implementation with vector support at [`MakieTeX.jl`](https://github.com/JuliaPlots/MakieTeX.jl).
+There is experimental support for using Plots.jl and RecipesBase.jl recipes at [MakieRecipes.jl](https://github.com/JuliaPlots/MakieRecipes.jl), and a prototype TeX plotting implementation with vector support at [MakieTeX.jl](https://github.com/JuliaPlots/MakieTeX.jl)
```
Comment (briochemc) [inline README.md:49]: Removed punctuation at the end here, too.

File: README.md [hunk 4]
```diff
@@ -45,99 +44,25 @@ sudo dnf install cmake libXrandr-devel libXinerama-devel libXcursor-devel
...
 
-Examples, and test infrastructure, are hosted at [`MakieGallery.jl`](https://github.com/JuliaPlots/MakieGallery.jl)
+Examples, and test infrastructure, are hosted at [MakieGallery.jl](https://github.com/JuliaPlots/MakieGallery.jl)
```
Comment (briochemc) [inline README.md:51]: There was no ending punctuation here before, but maybe you could add it :)

**Comment by asinghvi17** [comment]
Done, thanks!  I'll merge soon-ish if this looks right.

**Comment by briochemc** [comment]
(I'm sorry I'm very new to this reviewing thing so I don't really know what I'm doing. Hopefully it is helpful!)

**Pull request merged**